src/cmd/services/m3coordinator/handler/prometheus/remote/read_test.go

```diff
@@ -44,6 +44,11 @@ import (
 	"github.com/golang/snappy"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/uber-go/tally"
+)
+
+var (
+	promReadTestMetrics = newPromReadMetrics(tally.NewTestScope("", nil))
```

Comment:
nit: can we check that they're incremented as expected?

Reply:
I verified in my end-to-end testing on gcp that it works correctly. we might be able to do this type of testing when we add some integration tests?

Reply:
Can pass in a fakeMetrics which can track calls to each metric count?